install/paymenter-install.sh

```diff
@@ -0,0 +1,141 @@
+#!/usr/bin/env bash
+
+# Copyright (c) 2021-2025 community-scripts ORG
+# Author: Nícolas Pastorello (opastorello)
+# License: MIT | https://github.com/community-scripts/ProxmoxVE/raw/main/LICENSE
+
+source /dev/stdin <<<"$FUNCTIONS_FILE_PATH"
+color
+verb_ip6
+catch_errors
+setting_up_container
+network_check
+update_os
+
+msg_info "Installing Dependencies"
+$STD apt-get install -y \
+    curl \
+    git \
+    software-properties-common \
+    apt-transport-https \
+    ca-certificates \
+    gnupg
+$STD add-apt-repository -y ppa:ondrej/php
+$STD apt-get update -y
+$STD apt-get install -y \
+    php8.2 \
+    php8.2-{common,cli,gd,mysql,mbstring,bcmath,xml,fpm,curl,zip} \
+    mariadb-server \
+    nginx \
+    redis-server
```

Comment:
```suggestion
    curl \
    sudo \
    mc \
    git \
    software-properties-common \
    apt-transport-https \
    ca-certificates \
    gnupg \
    php8.2 \
    php8.2-{common,cli,gd,mysql,mbstring,bcmath,xml,fpm,curl,zip} \
    mariadb-server \
    nginx \
    redis-server
```
The ppa repo is not needed, 8.2 is in the debian 12 repos.